### PR TITLE
Featurization updates and utility functions

### PR DESCRIPTION
**Pull request by joebhakim**
This featurization code allows for a richer representation of input data for better downstream classification. This is a retroactive comment!!

### CODE REVIEW

File: features/features_utils/data_cleaning.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
+import pandas as pd
+import numpy as np
+
+def clean_static_df(static_df):
```
Comment (joebhakim) [inline features/features_utils/data_cleaning.py:4]: This should include a description of the schema expected as input here.

File: features/build_features.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    treat_df_clean = clean_treat_df(treat_df, static_df)
+    outcome_df_clean = clean_outcome_df(static_df)
+
+    static_df_clean.to_csv(output_filepath + '/static_vars.csv')
+    dynamic_df_clean.to_csv(output_filepath + '/dynamic_vars.csv')
+    treat_df_clean.to_csv(output_filepath + '/treat_vars.csv')
+    outcome_df_clean.to_csv(output_filepath + '/outcome_vars.csv')
+
+    print('getting CYXT tensor for downstream modeling...')
+    CXYT_df = get_CXYT_for_modeling(static_df_clean,
```
Comment (joebhakim) [inline features/build_features.py:42]: Here, we're using this CYXT data format as a generic way to do this representation for downstream modeling

**Review by joebhakim — COMMENTED** [review]
Left some minor notes here, this PR should be good to go.

File: features/build_features.py (hunk 2 above)
Comment (ErezKaminski) [reply features/build_features.py:42]: Great. That is what we wanted to do per the specification.

**Pull request merged**